trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-for-report-helper.ts

```diff
@@ -159,7 +166,7 @@ const getKeyingSheetAdjustmentViewModel = (adjustment: KeyingSheetAdjustment | n
 const mapKeyingSheetFeePaymentsToKeyingSheetFeePaymentsViewModel = (feePayments: KeyingSheetRow['feePayments']) =>
   feePayments.map(({ currency, amount, dateReceived }) => ({
     formattedCurrencyAndAmount: getFormattedCurrencyAndAmount({ currency, amount }),
-    formattedDateReceived: dateReceived ? format(new Date(dateReceived), 'd MMM yyyy') : undefined,
+    formattedDateReceived: dateReceived ? format(new Date(dateReceived), DATE_FORMAT.DAY_SHORT_MONTH_YEAR) : undefined,
```

Comment:
```suggestion
    formattedDateReceived: dateReceived && format(new Date(dateReceived), DATE_FORMAT.DAY_SHORT_MONTH_YEAR)
```

Reply:
This doesn't work because dateRecevied is of type `string | null` but formattedDateReceived is `string | undefined` and so we get type errors if we set it to `null`

Reply:
Could we perhaps align the 2x types? Then could simplify the above condition. Feels strange to have `string | null` and `string | undefined` for the essentially the same date field

Reply:
I think for consistency with other view model types I'd rather leave it as it is. They are actually quite different in that one is actually an date stamp and the other is the formatted date and I think it would be weird to have different types for our formatted date fields in different pages given we have a lot of them.